### ISSUE TITLE
Update install instructions in README

### ISSUE THREAD

**Issue by mcrowe** [zo0r/react-native-push-notification#380]
Following the install instructions verbatim won't work. `react-native link` requires the library to be in your `package.json`, and so it does nothing unless `react-native-push-notification` is saved to the `package.json`. To make things really clear, the instructions should be changed to:

```
npm install react-native-push-notification --save
react-native link
```

**Comment by sebo** [comment]
@mcrowe there is actually already a pull request https://github.com/zo0r/react-native-push-notification/pull/326 . I actually don't know why they wouldnt merge it...

**Comment by mcrowe** [comment]
Good stuff. I'll keep this issue open until that pull request is merged, as it seems to have been forgotten.

**Comment by sebo** [comment]
might be related to this issue https://github.com/zo0r/react-native-push-notification/issues/343

**Comment by igorsantos07** [comment]
The `--save` option is already there :)

**Comment by mcrowe** [comment]
Yup. Looks like this was fixed in commit e17d8a1f065dbe5f2f3ab864f0cf2e56bd778bc6, which is merged into master. Closing.
